packages/web/src/new/sdk/SolanaSDK/models/APIEndpoint.ts

```diff
@@ -69,4 +95,14 @@ export class APIEndpoint {
     }
     return url;
   }
+
+  toJSON(): APIEndpointProps {
```

Comment:
don't see place where its using

Reply:
it's used by `JSON.stringify` when save Defaults to LocalStorage here https://github.com/p2p-org/p2p-wallet-web/blob/b24d5ecfbe80d2d9199b114fd68ad86f17ba6876/packages/web/src/new/services/Defaults.ts#L21